### ISSUE TITLE
"code --install-extension example.vsix" does not work

### ISSUE THREAD

**Issue by gizmodus** [microsoft/vscode#115877]
- VSCode Version: 1.53.0
- OS Version: Windows 10

Steps to Reproduce:

1. Build an extension with `vsce package`
2. Install the extension with `code --install-extension example.vsix`
3. This command opens VSCode but the extension does not appear in the list of installed extensions

We have used the above command for months and it always worked. Since a few days / weeks it does not work anymore. Installing the extension inside VSCode via the command `Extensions: Install from VSIX` works normally.

Does this issue occur when all extensions are disabled?: Yes

**Comment by sandy081** [comment]
Seems like cli is not getting invoked.

**Comment by bpasero** [comment]
@sandy081 I cannot reproduce on macOS at least:

```
~/Development/Microsoft/monaco> code --install-extension example.vsix
Installing extensions...
ENOENT: no such file or directory, open '/Users/bpasero/Development/Microsoft/monaco/example.vsix'
Failed Installing Extensions: file:///Users/bpasero/Development/Microsoft/monaco/example.vsix
```

Can you please test windows? I don't see how my :lipstick: changes there could have any impact.

**Comment by sandy081** [comment]
Also works in windows.

@bpasero Are there any diagnosing steps user can follow to check whey CLI is not getting invoked?

**Comment by gizmodus** [comment]
@sandy081 @bpasero Thank you for the quick reactions! I just checked my environment variables and found that along the "bin" folder where the CLI seams to be located, the main path of VSCode is also defined under "PATH". Removing that folder solved the problem for me.
It's possible that I added this folder some time ago when I was experimenting with a script that opens several instances of VSCode in different folders. I am not sure though.
